casr/tests/tests.rs

```diff
@@ -2412,6 +2413,20 @@ fn test_casr_cluster_c() {
 
     assert_eq!(clusters_cnt, 9, "Clusters count mismatch.");
 
+    // Check crashline deduplication:
+    // 2.casrep and 20.caserp without crashlines => no dedup
+    // 3.casrep and 30.caserp with crashlines => dedup
```

Comment:
// all clusters have only 1 casrep, except the second one.

Reply:
Done